File: fix_auth.py
import fileinput
f = fileinput.input('C:/Users/STEMLENS/Music/Jumu AI/frontend/src/lib/AuthContext.tsx', inplace=True)
for line in f:
    print(line.replace('redirectTo: window.location.origin', 'redirectTo: window.location.origin + " /onboarding/basic-questions\'), end='')

File: fix_auth.js
const fs = require('fs');
const path = 'C:/Users/STEMLENS/Music/Jumu AI/frontend/src/lib/AuthContext.tsx';
let content = fs.readFileSync(path, 'utf8');
content = content.replace('redirectTo: window.location.origin', 'redirectTo: window.location.origin + " /onboarding/basic-questions\');
fs.writeFileSync(path, content);
console.log('AuthContext updated');
